docs/source/examples/zne_logical_rb_cirq_stim.md

```diff
@@ -0,0 +1,302 @@
+Fixing a sampling budget across unmitigated and mitigated workflows provides a fair comparison of the results in terms of the resources required.
+
+```{code-cell} ipython3
+def scale_shots(
+    num_device_qubits, scaled_distance, base_shots, n_qubits_circuit
+):
+    used_qubits = n_qubits_circuit * scaled_distance**2
+    return base_shots * int(num_device_qubits / used_qubits)
+```
+
+To minimize execution time, we convert each trial circuit only once, outside of the execution loop.
+
+```{code-cell} ipython3
+noisy_results = np.zeros((trials, len(d_array)))
+stim_circuits = [
+    stimcirq.cirq_circuit_to_stim_circuit(c) for c in filled_circuits
+]
+
+for di in range(len(d_array)):
+    for t in range(trials):
+        noisy_results[t, di] = stim_executor(
+            stim_circuits[t],
+            p_err,
+            p_th,
+            d_array[di],
+            scale_shots(device_size, d_array[di], 4 * base_shots, n_qubits),
+        )
+```
+
+Next we apply ZNE, with the noise scaling method of global unitary folding and scale factors {1, 3, 5, 7}, to each trial circuit and at each code distance simulated in the unmitigated case.
+Unlike other Mitiq examples, here the lower-level scaling and extrapolation functions are called explicitly.
+The circuits used here are deeper than those of other Mitiq examples and therefore take longer to fold and convert (sampling time is less of a concern with Stim), so we want to fold and convert the circuit as few times as possible.
+
+```{code-cell} ipython3
+scale_factors = [1, 3, 5, 7]
+fac = mitiq.zne.PolyFactory(scale_factors=scale_factors, order=3)
+scaling = mitiq.zne.scaling.folding.fold_global
+```
+
+```{code-cell} ipython3
+scaled_stim_circuits = []
+for c in filled_circuits:
+    scaled_stim_circuit = []
+    for s in scale_factors:
+        scaled_circuit = mitiq.zne.scaling.fold_global(c, s)
+        scaled_stim_circuit.append(
+            stimcirq.cirq_circuit_to_stim_circuit(scaled_circuit)
+        )
+    scaled_stim_circuits.append(scaled_stim_circuit)
+```
+
+```{code-cell} ipython3
+mitigated_results = np.zeros((trials, len(d_array)))
+
+for di in range(len(d_array)):
+    for t in range(trials):
+        fac.reset()
+        for s, f in zip(scale_factors, scaled_stim_circuits[t]):
+            fac.push(
+                {"scale_factor": s},
+                stim_executor(
+                    f,
+                    p_err,
+                    p_th,
+                    d_array[di],
+                    scale_shots(
+                        device_size, d_array[di], base_shots, n_qubits
+                    ),
+                ),
+            )
+        mitigated_results[t, di] = fac.reduce()
+```
+
+## Plot results
+
++++
+
+Finally, plot the mean unmitigated and ZNE-mitigated expectation values, averaged over the trial circuits, at every code distance simulated. The error bars indicate the standard deviation.
+
+```{code-cell} ipython3
+:tags: [remove-output]
```

Comment:
I think we can use `skip-execution` here if we aren't going to show output, right?

Reply:
We keep the `remove-output` tag to test that the full example runs w/o errors, while only showing the plot from the long circuits because that's where a fast stabilizer simulator like Stim is needed.